cms/search/tests/test_utils.py

```diff
@@ -0,0 +1,30 @@
+from django.test import TestCase
+from wagtail.models import Page
+
+from cms.articles.models import ArticleSeriesPage, ArticlesIndexPage
+from cms.articles.tests.factories import StatisticalArticlePageFactory
+from cms.search.utils import build_page_uri, get_model_by_name
+
+
+class SearchUtilsTests(TestCase):
```

Comment:
Perhaps worth some extra test cases for `build_page_uri` to ensure the first path element is always stripped correctly, regardless of what it is? Currently it is always `/home`, but that slug could be changed, e.g. translated for other languages.